command

Mboesn opened the pull request:
Made the set liftheight commands with pid

Mboesn committed:
command

Made the set liftheight commands with pid

esh4 commented on src/main/java/frc/robot/Commands/SetLiftHeight.java:
```diff
@@ -0,0 +1,58 @@
+  // Make this return true when this Command no longer needs to run execute()
+  @Override
+  protected boolean isFinished() {
+    return pidController.onTarget();
```
Check out the isFinished method in this command from spikesLib:
https://github.com/Spikes-2212-Programming-Guild/SpikesLib/blob/master/src/com/spikes2212/genericsubsystems/basicSubsystem/commands/MoveBasicSubsystemWithPID.java

esh4 commented on src/main/java/frc/robot/OI.java:
```diff
@@ -0,0 +1,15 @@
+import edu.wpi.first.wpilibj.buttons.JoystickButton;
+
+public class OI {
+    private XboxController xbox; 
```
delete the OI. It doesn't need to be implemented here.

Mboesn committed:
IsFinished in SetLiftHeight

yeah yeah, yeah yeah - we on da wayy up
yeah yeah, yeah yeah - we on da wayy up

yeah yeah, yeah yeah - we on da wayy up

yeah yeah, yeah yeah - we on da wayy up
yeah yeah, yeah yeah - we on da wayy up
yeah yeah, yeah yeah - we on da wayy up
yeah yeah, yeah yeah - we on da wayy up
yeah yeah, yeah yeah - we on da wayy up
yeah yeah, yeah yeah - we on da wayy up
yeah yeah, yeah yeah - we on da wayy up
yeah yeah, yeah yeah - we on da wayy up

esh4 commented on src/main/java/frc/robot/RobotConstants.java:
```diff
@@ -49,5 +50,9 @@
+     * the amount of time we want the lift to stay on the target fof the PID to end
+     */
+    public static final double LIFT_WAIT_TIME = 1;
 
```
Maybe this should be in the Command itself?

Mboesn committed:
lift wait time

mmmmm thats  taisstyyyyy mmmmmmmmmmmmmmmmmmm

esh4 committed:
Merge branch 'master' into LiftCommands
